content/personal/team.md

```diff
@@ -0,0 +1,317 @@
+---
+title: "The NeoForged Team"
+date: 2025-04-02T00:00:00+00:00
+draft: true
+categories:
+- Personal Blog
+author: neoforged
+summary: "The people behind the NeoForged project."
+description: "This article lists all current members of the NeoForged project alongside a short description."
+---
+
+Hello! In this article, we would like to introduce ourselves. Who are we, what do we do, and why are we a member of the project? For each team member, you can (hopefully) find an answer here.
+
+Members in all categories are ordered alphabetically by their Discord handle, which is given after the name.
+
+## Steering Council
+
+The Steering Council is what an admin team would be in most other projects. Its members are elected from among the team members, by the team members, on an annual basis.
+
+### gigaherz (@gigaherz)
+
+**GitHub profile**: https://github.com/gigaherz
+
+**Roles:** Steering Council, Moderator, Maintainer
+
+_No description provided._
+
+### Orion (@oriononline)
+
+**GitHub profile**: https://github.com/marchermans
+
+**Roles:** Steering Council, Moderator, Maintainer, Bot Team, Foundation Team, Infrastructure Team
+
+_No description provided._
+
+### TelepathicGrunt (@telepathicgrunt)
+
+**GitHub profile**: https://github.com/TelepathicGrunt
+
+**Roles:** Steering Council, Maintainer
+
+_No description provided._
```

Comment:
TelepathicGrunt is a maintainer that is focused mostly on the worldgen side of Minecraft as well as discussing tag usages. He also created [The Bumblezone](https://www.curseforge.com/minecraft/mc-mods/the-bumblezone-forge), [Repurposed Structures](https://www.curseforge.com/minecraft/mc-mods/repurposed-structures), [Structure Layout Optimizer](https://www.curseforge.com/minecraft/mc-mods/structure-layout-optimizer), and quite a few more mods too.

Outside of Minecraft, he plays video games and eats breadsticks.